interfaces/builtin/udisks2.go

```diff
@@ -37,7 +38,9 @@ const udisks2BaseDeclarationSlots = `
     allow-installation:
       slot-snap-type:
         - app
-    deny-connection: true
+        - core
```

Comment:
I must say that the syntax, while initially cryptic, has grown on me. This is very nice :)